src/Modbus_RTU.ino

```diff
@@ -457,26 +457,37 @@ struct ModbusRTU_struct  {
       // Send the byte array
       startWrite();
       easySerial->write(_sendframe, _sendframe_used);
+      // drop all data from buffer
+      easySerial->flush();
```

Comment:
Flush first, then write, then start to read.

Reply:
it should be ok:
on hw serial -> every write provide flush
on sw serial -> every byte is sent with blocking

Reply:
Never mind, I thought flush was to clear the read buffer...